### DISCUSSION TITLE
Structure in the TeX panel is empty

### DISCUSSION THREAD

**Discussion by jademath** [James-Yu/LaTeX-Workshop#3865] [Q&A]
I just installed it and checked the setting `latex-workshop.view.outline.sections`. It has the default values and the folding works. But the `Structure` on the TeX panel is completely empty.

**Comment by slyandsmart** [comment]
I have the same problem, but without installing anyting latex related. I just playd with ChatGPT Extentions. But uninstalled all of them. 
Now my Structure panel is empty. Everything else works quite fine.

**Comment by Tedxz** [comment]
I have the same problem in v9.11.0. 

In the LaTeX-Workshop output, it continuously outputs
`[Structure] Structure updated with 0 root sections for %WS1%/xxx.tex .`

Going back to v9.10.0 solves the problem temporarily.

**Comment by jlelong** [comment]
This is a bug #3887
